tests/st/runtime/test_dyn_orch_shape.py

```
@@ -747,87 +746,81 @@ def compute_expected(self, tensors, params=None):
 class TestDynOrchShapeOperations:
     """Test suite for dynamic orchestration shape operations."""
 
-    @pytest.mark.parametrize("backend", PLATFORMS)
+    @pytest.mark.parametrize("platform", PLATFORMS)
     @pytest.mark.parametrize("shape", _DYN_SHAPES)
-    def test_dyn_orch_add(self, test_runner, shape, backend):
+    def test_dyn_orch_add(self, test_runner, shape, platform):
         """Test add where both InCore and orchestration use dynamic M×N dims."""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)